### ISSUE TITLE
[BUG] ENOENT: no such file or directory cache.json

### ISSUE THREAD

**Issue by Tejaswgupta** [gptlint/gptlint#18]
I'm using Llama-3-instruct running on VLLM server. I've tried installing both with npm and pnpm and setting a local cache directory but can't get past this issue.
```
tejasw@Tejasws-iMac-M1 votum % npx gptlint --api-base-url http://xxx.xxx.xxx:8083/v1 --api-key "empty" --model casperhansen/llama-3-70b-instruct-awq --weak-model casperhansen/llama-3-70b-instruct-awq "src/app/home/tasks/[taskId]/component.tsx" --no-cache
rule "effective-tsconfig" unexpected preProcess error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
Linter tasks {
  numFiles: 1,
  numRules: 14,
  numTasks: 10,
  numTasksFilteredPrecheck: 1,
  numTasksFilteredGrit: 2
}
rule "use-correct-english" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "always-handle-promises" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "avoid-type-info-in-docs" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "no-hardcoded-secrets" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "consistent-identifier-casing" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "prefer-array-at-negative-indexing" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "liberal-accept-strict-produce" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "soc2-no-leak-user-data" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
rule "prefer-types-always-valid-states" file "src/app/home/tasks/[taskId]/component.tsx" unexpected error: ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
✔ Preprocessing 1 files and 14 rules
❯ Linting src/app/home/tasks/[taskId]/component.tsx
  ✖ consistent-identifier-casing
    → ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
  ✖ prefer-types-always-valid-states
    → ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
  ⠇ semantic-variable-names
  ✖ soc2-no-leak-user-data
    → ENOENT: no such file or directory, open '/Users/tejasw/Downloads/votum/node_modules/.cache/gptlint/cache.json'
```

**Comment by transitive-bullshit** [comment]
Interesting. I'm not able to repro the cache issue, but I have a general idea of what might be going wrong.

I just pushed a release which will give a lot more info for debugging bugs like this in the future. Would you mind upgrading to [1.5.3](https://github.com/gptlint/gptlint/releases/tag/v1.5.3) and re-running this w/ the output? That should help me a lot with narrowing down what might be going wrong.

Thanks for the bug report!
